src/pages/getting-started/index.mdx

```diff
@@ -38,121 +62,106 @@ From a more tech perspective, the LiveChat Platform provides:
 
-If you want to know more about extending the LiveChat UI, check out [this guide](https://developers.livechat.com/docs/building-apps-for-livechat/#extending-user-interfaces).
+- [Agent Chat API](/messaging/agent-chat-api/)
+- [Customer Chat API](/messaging/customer-chat-api/)
 
-# Platform APIs
+Together, the [Agent Chat API](/messaging/agent-chat-api/) and the [Customer Chat API](/messaging/customer-chat-api/) create the <a href="https://developers.livechat.com/docs/messaging" target="_blank">Messaging APIs</a>. It is a full set of methods to use either as an agent or as a customer. We divide it though, to make it more use-case-specific. The distinction is intuitive; when you want to join a chat as an agent, use the Agent Chat API. To interact as a customer, use the Customer Chat API. 
 
-Many LiveChat and HelpDesk applications built upon the LiveChat Platform need to use our API. For that reason, we make it accessible and let developers play with it.
+These two sets of APIs are very versatile. The Agent Chat API lets you build your custom Agent App, as well as browser chat archives, or ban customers. With the use of the Customer Chat API, you can build a custom Chat Widget. Both APIs allow for interactions, such as joining a chat, posting messages, etc.
 
-There are two **messaging APIs** for LiveChat:
+If Messaging APIs seem overwhelming, or you don't have time to play with them, <a href="https://developers.livechat.com/docs/messaging/chat-sdk" target="_blank">Chat SDK</a> comes to help you. It was designed to allow developers to build apps quickly. It comes with a method template for sending plain text messages. Based on that method, you can easily build new methods.
 
-- [Agent Chat API](/messaging/agent-chat-api/)
-- [Customer Chat API](/messaging/customer-chat-api/)
+Platform also provides two Software Development Kits, <a href="https://developers.livechat.com/docs/messaging/go-sdk" target="_blank">LiveChat Go SDK</a>, written in Go, and <a href="https://developers.livechat.com/docs/messaging/js-sdk" target="_blank">LiveChat JS SDK</a>, written in Typescript. They are tools for developers to build external backend apps which extend LiveChat features. The SDKs make it easy to use the [Agent Chat API](/messaging/agent-chat-api/), the [Customer Chat API](/messaging/customer-chat-api/), and the [Configuration API](/management/configuration-api/).
 
-There's also the [Configuration API](/management/configuration-api/).
+# Management
 
-Public API for HelpDesk is widely described in the <a href="https://api.helpdesk.com/docs#section/Introduction" target="_blank">HelpDesk API Documentation</a>.
+The <a href="https://developers.livechat.com/docs/management" target="_blank">Configuration API</a> is a service for storing license configurations. It can be used to build complex integrations or to configure resources directly via the API.
 
-## Chat API
+Configuration API allows:
```

Comment:
```suggestion
The Configuration API allows for:
```

Reply:
Idk, but the bare "allows" sounds a bit like "gives permission" to me

Reply:
I was actually deleting it and adding xd But now that you wrote it, yes it might be a bit like gives permission" to me